atc/db/pipeline.go

```diff
@@ -50,7 +50,10 @@ type Pipeline interface {
 	GetBuildsWithVersionAsInput(int, int) ([]Build, error)
 	GetBuildsWithVersionAsOutput(int, int) ([]Build, error)
 	Builds(page Page) ([]Build, Pagination, error)
+
 	CreateOneOffBuild() (Build, error)
+	CreateStartedBuild(plan atc.Plan) (Build, error)
```

Comment:
Do we still need `CreateOneOffBuild`?

Reply:
ugh, yes. its used absolutely everywhere in the tests and I did not want to mess with that.